src/app/shared/licenses/licenses.service.ts

```diff
@@ -1,16 +1,20 @@
 import { Injectable } from '@angular/core';
 import { License } from './licenses.model';
 import { HttpClient } from '@angular/common/http';
-import { shareReplay } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
 
 @Injectable({
   providedIn: 'root',
 })
 export class LicensesService {
-  licenses$ = this.http.get<License[]>('/assets/licenses.json').pipe(
-    shareReplay(),
-  );
+
+  public licenses$: BehaviorSubject<License[]> = new BehaviorSubject<License[]>(null);
+
+  private readonly LICENSCE_URL = '/assets/licenses.json';
```

Comment:
typo: `LICENSCE` -> `LICENSE`
Same with other occurence(s)
```suggestion
  private readonly LICENSE_URL = '/assets/licenses.json';
```